script/glam/generate_fenix_sql

```diff
@@ -10,19 +10,38 @@ dataset=org_mozilla_fenix_stable
 tables=$(bq ls --format=json $project:$dataset | \
         jq -r '.[] | .tableReference.tableId')
 
-function write_sql {
+function write_scalars {
     local table=$1
     local directory="sql/glam_etl/fenix_clients_daily_scalar_aggregates_${table}"
     mkdir -p "$directory"
-    python3 -m bigquery_etl.glam.clients_daily_scalar_aggregates \
+    if ! python3 -m bigquery_etl.glam.clients_daily_scalar_aggregates \
         --source-table "$dataset.$table" \
-        > "$directory/query.sql"
-    echo "generated $directory/query.sql"
+        > "$directory/query.sql"; then
+            echo "skipping $directory/query.sql: no probes found"
+            rm -r "$directory"
+    else
+        echo "generated $directory/query.sql"
+    fi
+}
+
+function write_histograms {
+    local table=$1
+    local directory="sql/glam_etl/fenix_clients_daily_histogram_aggregates_${table}"
+    mkdir -p "$directory"
+    if ! python3 -m bigquery_etl.glam.clients_daily_histogram_aggregates \
+        --source-table "$dataset.$table" \
+        > "$directory/query.sql"; then
+            echo "skipping $directory/query.sql: no probes found"
+            rm -r "$directory"
```

Comment:
This logic is useful for avoiding scans over tables where there are no distributions.